app/features/atp/fraud-check/ui/idx/view.njk

```diff
@@ -0,0 +1,41 @@
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/select/macro.njk" import govukSelect %}
+{% from "components/languageLink/macro.njk" import languageLink %}
+{% from "govuk/components/fieldset/macro.njk" import govukFieldset %}
+{% from "govuk/components/date-input/macro.njk" import govukDateInput %}
+{% from "components/autoInput/list/macro.njk" import autoList %}
+{% from "components/autoInput/fill/macro.njk" import autoFill %}
+{% from "components/autoInput/arrow/macro.njk" import autoArrow %}
+{% from "govuk/components/textarea/macro.njk" import govukTextarea %}
+{% extends "common/layout/view.njk" %}
+{% set pageTitleName = 'pages.ipv.identity-evidence.title' | translate %}
+{% block content %}
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            {% set errorList = errorList | lastErrors(["#dob"]) %}
+            {% include "components/errorSummary/errorSummary.njk" %}
+        <h1 class="govuk-heading-xl">Please enter the following information</h1>
+            <p>Please enter the following information</p>
+            <form action="/fraud-check" method="post" class="form" novalidate>
+                <input type="hidden" name="_csrf" value="{{csrfToken}}"/>
+                {{ govukTextarea({
+                    name: "fraudCheckData",
+                    id: "fraudCheckData",
+                    label: {
+                        text: "Fraud Check Data",
+                        classes: "govuk-label--l",
+                        isPageHeading: true
+                    },
+                    hint: {
+                        text: "Enter fraud check data as JSON"
```

Comment:
Do we know anything about what this fraud data is going to look like? If its constrained somehow, it would be nice to have that in the UI so that less API knowledge is required to experiment with the fraud settings.

Reply:
Not at the moment.